pyvista/plotting/qt_plotting.py

```diff
@@ -323,10 +327,15 @@ class QVTKRenderWindowInteractorAdapter(QObject):
 
     def __init__(self, parent, **kwargs):
         """Initialize the internal interactor."""
+        try:
+            assert_empty_kwargs(**kwargs)
+        except TypeError as e:
+            # user doesn't want to know about __init__
+            raise TypeError(str(e).split('for `')[0]) from None
         self.interactor = QVTKRenderWindowInteractor(parent=parent)
         self.interactor.dragEnterEvent = self.dragEnterEvent
         self.interactor.dropEvent = self.dropEvent
-        super(QVTKRenderWindowInteractorAdapter, self).__init__(**kwargs)
+        super(QVTKRenderWindowInteractorAdapter, self).__init__()
```

Comment:
Good point.  I actually tried out shape, border the other day and as nothing happened, I had to assert empty keyword args.

We may consider adding tests for these features.  It won't improve code coverage as a whole as these features will be covered by alternative tests.  If the `qt` portion keeps getting bigger, we might consider isolating it and creating a new module.

Reply:
As far as I understood, the goal of `pyvista-gui` is to be the powerhouse and the `BackgroundPlotter` is the minimal interactive plotter.

+1 to isolate the big qt parts and I would vote to keep the `BackgroundPlotter` as light as possible with the bare minimum.

Reply:
@banesullivan and @GuillaumeFavelier, we were considering adding additional features to `BackgroundPlotter` like adding in a tree view.  Should we simply move these to the `pyvista-gui`?  Having a Paraview-esque GUI might be the direction we can move `pyvista-gui` in, rather than simply making it a plug and play module for other programs.

With my closed source project, there are too many variations and some of them require monkey patching the GUI to such a degree that it creates extra work.  Why not go with @GuillaumeFavelier 's suggestion and have advanced functionality for the BackgroundPlotter in `pyvista-gui`.

To tell you the truth, with the problems we had with `BackgroundPlotter`, I'm half tempted to move it all to `pyvista-gui`.  We can then control the `PyQt5` version and do better testing there.  `pyvista` is becoming a bit of a monster these days and it might be better to split it up.

@pyvista/developers, thoughts?  Might open a separate thread on slack/gitter...

Reply:
Nevermind, opening a new issue.

@GuillaumeFavelier, fixed the kwargs.  Adding `shape=` now works as expected.

Reply:
I'll fix these kwargs and chine in on #614